testing/trino-tests/src/test/java/io/trino/security/TestFunctionsInViewsWithFileBasedSystemAccessControl.java

```diff
@@ -47,13 +45,12 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         String securityConfigFile = getResource("file-based-system-functions-access.json").getPath();
-        SystemAccessControl accessControl = new FileBasedSystemAccessControl.Factory().create(ImmutableMap.of(SECURITY_CONFIG_FILE, securityConfigFile));
         DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(testSessionBuilder()
                         .setCatalog(Optional.empty())
                         .setSchema(Optional.empty())
                         .build())
                 .setNodeCount(1)
-                .setSystemAccessControl(accessControl)
+                .setSystemAccessControl("file", Map.of(SECURITY_CONFIG_FILE, securityConfigFile))
```

Comment:
It is so nice we can pass config like for a connector

Reply:
thanks!